Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -741,8 +745,16 @@ - (BOOL)_isHostedInAnotherProcess;
     [contentView() _showPlaybackTargetPicker:hasVideo fromRect:elementRect routeSharingPolicy:policy routingContextUID:contextUID.createNSString().get()];
 }
 
-bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy* proxy, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
 {
+    RELEASE_LOG_INFO(WebRTC, "PageClientImpl::handleRunOpenPanel");
```

Comment:
It's strange to see this in the `WebRTC` category, since this method is more generally used for file/photo upload. But I don't have another suggestion from the existing set of categories.

Reply:
Done, thanks